src/components/Firestore/Requests/utils.ts

```diff
@@ -14,6 +14,25 @@
  * limitations under the License.
  */
 
+import { OutcomeData } from './types';
+
+// Matches the material-icon name by request outcome
+const ICON_SELECTOR = {
```

Comment:
Let's make this a `const enum`:

```ts
const enum OutcomeIcons {
  ALLOW = 'check_circle',
  // ...
};

// Then:
{icon: OutcomeIcons.ALLOW}
```

Reply:
I think this constant should stay a dictionary, another component uses this dictionary to get the correct icon based on the outcome string, that code is just not part of this PR.

Reply:
Ah okay, that's fine then